pallets/streaming/src/types.rs

```diff
@@ -14,6 +14,8 @@ pub enum StreamStatus {
     Completed,
```

Comment:
should put inside Ongoing

Reply:
Why do it, I don't understand

Reply:
how about Ongoing { asCollateral: false|true }

Reply:
okay